README.md

```diff
@@ -41,8 +41,8 @@ A tarball containing everything needed for development and testing
 of the platform at [packages.claroline.net/releases][releases]. This is the
 fastest way to get started:
 
-    curl packages.claroline.net/releases/latest/claroline-6.x.x-dev.tar.gz | tar xzv
-    cd claroline-6.x.x-dev
+    curl packages.claroline.net/releases/latest/claroline-7.0.0-rc1.tar.gz | tar xzv
```

Comment:
If we use an explicit version number in the instructions, we'll have to update it at each release. No big deal, but we'll have to remember it.

The version is missing the env part though (dev/prod). In this case, it should probably be `7.0.0-RC1-dev`.